csv_generator/consumers/manuscript_consumer.py

```diff
@@ -26,7 +34,12 @@ def process(self, soup: BeautifulSoup, xml_file_name: str) -> None:
         :param xml_file_name:
         :return:
         """
-        msid = self.get_msid(soup)
+        if not soup.find('manuscript'):
```

Comment:
Is ok for this case, if any more logic is to be added then it could probably be moved to a `pre_process` method.

Reply:
What would the `pre_process` method do? Would it return a boolean?

Reply:
Yes, return a `bool`, it could handle any pre-processing work that has to pass in order to allow the process to be run.

But as I said, it is probably fine for now, but if the list of things that need to be`present` before processing increases then we should probably think about it.

Reply:
Yes, something re-usable for history and staging would be good. One thing that bugs me a bit is that manuscript and history are processed side by side and would report a warning on the same thing multiple times (depending on the number of consumers). That is why I thought maybe it should be moved up the chain.